panel/src/components/Forms/Input/TextareaInput.vue

```diff
@@ -104,6 +104,9 @@ export default {
 	watch: {
 		value() {
 			this.onInvalid();
+			this.$nextTick(() => {
+				this.$refs.input.autosize();
```

Comment:
I'm not sure if this also fixes the jumping issue, but it definitely will fix textareas that might receive updates from somewhere else through reactive changes.